### PR TITLE
docs: update some cassandane docs

### PR DESCRIPTION
**Pull request by elliefm**
What it says on the tin

### CODE REVIEW

File: cassandane/doc/running_tests.txt [hunk 1]
```diff
@@ -16,146 +16,103 @@ Cassandane tests are run out of the Cassandane directory itself, without
...
+All runtime state is created under the cassandane rootdir configured in
+cassandane.ini (by default: /var/tmp/cass).
 
-Cassandane (or more precisely, the Cyrus code it exercises) needs to be run
-either as the superuser or as the "cyrus" user.  If run as another user,
-it attempts to use 'sudo' to re-run itself as the "cyrus" user.
+Internally, Cassandane (or more precisely, the Cyrus code it exercises) needs
+to be run either as the superuser or as the "cyrus" user.  But you should
+generally invoke Cassandane as yourself, not as "cyrus" or "root".  It will
+to re-run itself using sudo, which you already configured during setup (didn't
```
Comment (lerlacher-fm) [inline cassandane/doc/running_tests.txt:25]: That sentence doesn't scan for me - should that "to" be a "then"?

Comment (elliefm) [reply cassandane/doc/running_tests.txt:25]: "it will _try_ to", doh, thanks!

Comment (elliefm) [reply cassandane/doc/running_tests.txt]: fixed and force-pushed :)

**Review by ksmurchison — APPROVED** [review]
LGTM

**Pull request merged**